src/game/model/const.js

```diff
@@ -12,6 +12,7 @@ export const FPS = 60;
 // export const IS_HIDPI = window.devicePixelRatio > 1;
 export const IS_HIDPI = true // Set high resolution by default
 export const IS_IOS = /iPad|iPhone|iPod/.test(window.navigator.platform);
-export const IS_MOBILE = /Android/.test(window.navigator.userAgent) || IS_IOS;
+export const IS_ANDROID = /Android/.test(window.navigator.userAgent);
```

Comment:
Not used at the moment, but just keeping it for reference. Was part of `IS_MOBILE` before. Breaking them out provides for a better overview.